extensions/wikia/UserRenameTool/RenameUserProcess.class.php

```diff
@@ -560,324 +464,7 @@ private function doRun() {
 		}
 
 		$this->invalidateUser( $this->mOldUsername );
-
-		// process global tables
-		$this->addLog( "Initializing update of global shared DB's." );
-		$this->updateGlobal();
-
-		$callParams = array(
-			'requestor_id' => $this->mRequestorId,
-			'requestor_name' => $this->mRequestorName,
-			'rename_user_id' => $this->mUserId,
-			'rename_old_name' => $this->mOldUsername,
-			'rename_new_name' => $this->mNewUsername,
-			'rename_fake_user_id' => $this->mFakeUserId,
-			'phalanx_block_id' => $this->mPhalanxBlockId,
-			'reason' => $this->mReason,
-			'notify_renamed' => $this->mNotifyUser,
-		);
-		$task = ( new UserRenameTask() )
-			->setPriority( \Wikia\Tasks\Queues\PriorityQueue::NAME );
-		$task->call( 'renameUser', $wikiIDs, $callParams );
-		$this->mUserRenameTaskId = $task->queue();
-
-		return true;
-	}
-
-	/**
-	 * Processes shared database (wikicities) and makes all needed changes
-	 */
-	public function updateGlobal() {
```

Comment:
Where does the rename actually happen right now? :)

Reply:
in `renameAccount()`